src/build.py

```diff
@@ -1258,6 +1264,9 @@ def Summary(repos):
   print 'Failed steps: %s.' % buildbot.Failed()
   for step in buildbot.FailedList():
     print '    %s' % step
+  print 'Warned steps: %s.' % buildbot.Warned()
+  for step in buildbot.WarnedList():
+    print '    %s' % step
 
   if IsBuildbot():
     latest_file = '%s/%s' % (BUILDBOT_BUILDERNAME, 'latest.json')
```

Comment:
We should also maybe not update LKGR if there were warned steps (see L1275 below). I seemed to recall @jfbastien wanting that on #177. It's a little ambiguous about what LKGR actually means, especially since the different OS bots test some different things, but it seems fine to require that JSC works for now.